Ankit | Test | design issue on responsive screen

ankitdubeyoffice opened the pull request:
https://app.clickup.com/t/86d10kmh9

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **Style**
  * Adjusted contact preview list item heights for a one-line display.
  * Made import button status expand to full width on smaller viewports.
  * Simplified and improved responsive header layout and spacing for GST screens; date picker moved inline.
  * Enabled normal wrapping for customer names in invoice tables for better readability.

<sub>✏️ Tip: You can customize this high-level summary in your review settings.</sub>
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

ankitdubeyoffice committed:
design issue on responsive screen

dvCodeWorld commented on apps/web-giddh/src/app/downloads/components/imports/imports.component.scss:
```diff
@@ -19,6 +19,12 @@
         max-width: 160px;
         text-align: center;
```
```suggestion
        text-align: center;
        
        @include media-breakpoint-down("small-desktop") {
            min-width: 100%;
        }
```

dvCodeWorld commented on apps/web-giddh/src/app/downloads/components/imports/imports.component.scss:
```diff
@@ -19,6 +19,12 @@
     }
+    
+    @media (max-width: 1500px) {
+        .import-button-status {
```
remove

dvCodeWorld commented on apps/web-giddh/src/app/new-vs-old-Invoices/sales-bifurcation-details/sales-bifurcation-details.component.scss:
```diff
@@ -6,3 +6,7 @@
 
 }
+
+.white-space-normal {
```
```suggestion
// Will remove in prod pr
.white-space-normal {
```

ankitdubeyoffice committed:
style: fix responsive layout for import button status

Moved media query inside import-button-status class and updated to use SCSS mixin for better maintainability

ankitdubeyoffice committed:
style: update import button status breakpoint to large-desktop
